sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamAggregationTransforms.java

```diff
@@ -185,6 +185,7 @@ public AggregationAdaptor(
             aggregators.add(BeamBuiltinAggregations.createMin(call.type.getSqlTypeName()));
             break;
           case "SUM":
+          case "$SUM0":
```

Comment:
This one raises the issue that we maybe should start our test coverage of operators with a bunch of these internal variants that Beam SQL may already support, at least partly.

(this one is probably a hack)

Reply:
I agree. Also, should we support a separated built-in `sum0` function in the future? I think `sum` can't convert none element to zero automatically.